### ISSUE TITLE
Custom Filters?

### ISSUE THREAD

**Issue by gehrleib** [vladimir-tutin/Plex-Auto-Collections#17]
Would it be possible to do filters like...

```
  Comedy:
    genre: Comedy but not if it includes Romance
  Romantic Comedy:
    genre: Romance AND Comedy
```

**Comment by burkasaurusrex** [comment]
I don't believe this possible, primarily due to some limitations of Plex. This script uses Plex's search to select movies. When you perform a search or a filter in Plex, Plex ANDs together different filters (e.g. Year >= 1990 AND Critic Rating >= 7.0). Unfortunately, when you do multiple of the same filters, in Plex, Plex ORs together multiple filters (e.g. Genre = Comedy OR Genre = Romance).

tldr; you can create a collection with this script for any view you can filter your movie library down to in the web UI + a couple more (subfilters based on audio language, subtitle language and video-resolution).

**Comment by meisnate12** [comment]
i think you might be able to accomplish `Romantic Comedy` right now by doing

```
  Romantic Comedy:
    genre: Romance
    subfilters:
        genre: Comedy
```

also we might be able to add a not flag to allow Comedy but not Romantic

```
  Not Romantic Comedy:
    genre: Comedy
    subfilters:
        genre!: Romance
```

**Comment by meisnate12** [comment]
actually, after extensive research, this is possible right now just do this

```
  Not Romantic Comedy:
    genre!: Romance
    subfilters:
        genre: Comedy
```
